spyderlib/widgets/sourcecode/base.py

```diff
@@ -61,15 +63,26 @@ def setup_appearance(self, size, font):
         self.setFont(font)
         
     def show_list(self, completion_list, automatic=True):
+        types = [c[1] for c in completion_list]
+        completion_list = [c[0] for c in completion_list]
         if len(completion_list) == 1 and not automatic:
             self.textedit.insert_completion(completion_list[0])
             return
-        
+
         self.completion_list = completion_list
         self.clear()
-        self.addItems(completion_list)
+
+        lut = dict(instance='attribute.png', method='method.png',

```

Comment:
What this `lut` variable means? I mean, could you find a better name for it? Or at least a comment above about why you decided to name it like that?